CRM/CiviAwards/Service/AwardImport.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+use CRM_CiviAwards_Helper_CaseTypeCategory as CaseTypeCategory;
+
+/**
+ * Utility class for importing Awards using CSV importer extension.
+ */
+class CRM_CiviAwards_Service_AwardImport {
+
+  /**
+   * Create an Award and the related entities.
+   *
+   * @param array $params
+   *   Information for creating the award and related entities.
+   */
+  public function create(array $params) {
+    $tx = new CRM_Core_Transaction();
+
+    $params['case_type_category'] = $this->getCaseCategoryValueForAwards();
+    $params['definition'] = $this->createDefinition($params);
+    try {
+      $caseType = civicrm_api3('CaseType', 'create', $params);
+    }
+    catch (Exception $exception) {
+      $tx->rollback();
+
+      throw new API_Exception('Exception while saving the Case Type for Award: ' . $exception->getMessage());
+    }
+    $caseTypeId = array_shift($caseType['values'])['id'];
+
+    $params['case_type_id'] = $caseTypeId;
+    $params['award_manager'] = !empty($params['award_manager']) ? json_decode($params['award_manager'], TRUE) : [];
```

Comment:
I think it will be more intuitive to allow a comma separated list of contact ids for the award_manager column. This is easier to add for a lay user than adding a json string in the csv.

If we have that. Then we can do

```php
$params['award_manager'] = !empty($params['award_manager']) ? explode(",", $params['award_manager']) : [];
```

Reply:
I agree with that. Done here: https://github.com/compucorp/uk.co.compucorp.civiawards/pull/164/commits/98f1b35279127066c2ac190147e62247aabb22b4